controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -191,7 +191,7 @@ module.exports = async (req, res) => {
     //END LOCATIONS
 
     //OTHER USERS
-    if (page == 1 && topics) {
+    if (topics) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Removing the condition related to the page number might affect the pagination logic. If this condition was previously used to only fetch other popular users on the first page, its removal could mean that these users are fetched on every page, which may not be the intended behavior. Please ensure that this change aligns with your pagination strategy.

```diff
- if (page == 1 && topics) {
+ if (topics) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->